api/src/main/resources/absorb.yaml

```diff
@@ -0,0 +1,255 @@
+# This spec is mostly manually adapted from the Swagger 2 spec corresponding to each
+# page of the docs. These can be found through Chrome's network tab while loading
+# one of the pages.
+# Example: https://aoudev.myabsorb.com/Areas/ApiDocumentation/V15/Data/RestAuthentication.json
+# However, these specs were imperfect and needed to be manually edited.
+
+openapi: 3.0.1
+info:
+  title: Absorb
+  description: "Service for Absorb Interactions. API documentation can be found at https://aoudev.myabsorb.com/v1_5-doc"
```

Comment:
Do you have opinions about when to quote? This file uses double quotes more often than others I've seen. I value concision, so my general style recommendations look something like, "Do X only when necessary."

Reply:
Personally, always. Except when I forget.

From the article at https://news.ycombinator.com/item?id=34351503:

> Many of the problems with yaml are caused by unquoted things that look like strings but behave differently. This is easy to avoid: always quote all strings.

In re-reading this, turns out _keys_ are also affected, which I had forgotten. However, I'd think (hope?) that OpenAPI spec keys are avoiding these problems.

I'm eagerly waiting for the day when a "JSON with comments" or equivalent becomes mainstream.